convert2rhel/subscription.py

```diff
@@ -64,6 +68,16 @@ class StopRhsmError(Exception):
     """Raised with problems stopping the rhsm daemon."""
 
 
+class FileNotFoundError(OSError):
+    """Raised when the requested file cannot be found."""
+
+    pass
+
+
+class JSONDecodeError(ValueError):
+    pass
```

Comment:
Why do we need to recreate exceptions that already exist? Is this because python 2 or something similar?

Reply:
Yes for centos7 the tests were failing for those exceptions, but it had something to do with me using tmp_path instead of tmp_dir. Tests are passing now so i'm removing them